tests/api2/test_305_container.py

```diff
@@ -497,20 +495,20 @@ def test_38_verify_ix_chart_config_volumes(request):
     results = GET(f'/chart/release/id/{tc_release_id}/')
     assert results.status_code == 200, results.text
     assert isinstance(results.json(), dict), results.text
-    payload_volume = payload['values']['volume'][0]
-    assert payload_volume in results.json()['config']['volume'], results.text
+    payload_volume = payload['values']['volumes'][0]
+    assert payload_volume in results.json()['config']['volumes'], results.text
 
 
 @pytest.mark.dependency(name='tc_hostPathVolumes')
 def test_39_set_truecommand_ix_chart_hostPathVolumes(request):
-    depends(request, ['tc_chart_release'])
+    depends(request, ['tc_chart_release', 'hostpath-dataset'])
     global payload
     payload = {
         'values': {
             'hostPathVolumes': [
                 {
                     'hostPath': f'/mnt/{pool_name}/tc-hostpath',
-                    'mountPath': f'/mnt/{pool_name}/tc-mountpath',
+                    'mountPath': f'/mnt/{pool_name}/ix-applications/releases/truecommand/volumes/tc-mountpath',
```

Comment:
I am not sure I follow ? `mountPath` is inside the pod/container

Reply:
I have not idea where is the pod/container don't forget I am just learning about all those things.

Reply:
@ericbsd please feel free to ping me for any queries as always. Thanks